### DISCUSSION TITLE
The 3d object is flipped when using renderTexture

### DISCUSSION THREAD

**Discussion by ChristieLukose** [jnsmalm/pixi3d#186] [Q&A] [answered]
![without rendertexture](https://github.com/jnsmalm/pixi3d/assets/101198906/86d4750e-1755-406c-8b0f-b841388db3fe)
When rendered without render texture the 3d cube is placed perfectly with the text as Sprite3D on it,

![asRendertexture](https://github.com/jnsmalm/pixi3d/assets/101198906/585b0cf7-fee9-4ee7-8823-d848614f846c)
But the cube position and the texture and even the cube face is inverted when used as rendertexture.

[sample code](https://codesandbox.io/s/pixi3d-rendertextureissue-yxmg6g?file=/src/index.js)
Please help me solve this problem!!

**Comment by jnsmalm** [comment, marked as answer]
Can you try using the CompositeSprite instead? https://api.pixi3d.org/classes/CompositeSprite.html

**Comment by ChristieLukose** [comment]
Thank you it worked !!
